test/new-e2e/tests/cws/fargate_test.go

```diff
@@ -280,6 +280,11 @@ func getAgentFullImagePath(e *configCommon.CommonEnvironment) string {
 	if fullImagePath := e.AgentFullImagePath(); fullImagePath != "" {
 		return fullImagePath
 	}
+
+	if e.PipelineID() != "" && e.CommitSHA() != "" {
+		return fmt.Sprintf("669783387624.dkr.ecr.us-east-1.amazonaws.com/agent:%s-%s", e.PipelineID(), e.CommitSHA())
+	}
+
 	return agentDefaultImagePath
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
We should probably expose this method on test-infra-side to re-use it

Reply:
I think that's a good idea, since this on is just a copy/paste of the one we have on `test-infra`